README.md

```diff
@@ -587,15 +591,4 @@ API descriptions:
  * [OpenAPI](server/src/main/resources/openapi/reservoir.yaml)
  * [Schemas](server/src/main/resources/openapi/schemas/)
 
-Generated [API documentation](https://s3.amazonaws.com/indexdata-docs/api/reservoir/reservoir.html).
```

Comment:
This link to generated API documentation is still relevant. It is the new Index Data docs.